public/wp-content/themes/glific/src/js/main.js

```diff
@@ -26,9 +26,27 @@ jQuery(document).ready(function() {
 		jQuery(target).addClass('d-flex').removeClass('d-none');
 	});
 
-	jQuery('.show-more-blogs').on('click', show_more_blogs);
+	jQuery('.desktop-menu .menu-item-has-children').hover(function() {
```

Comment:
@mohitgusain Could we have done this with CSS? Any thoughts on using these two different approaches?

Reply:
@mohitgusain any update on this?

Reply:
Ok.